core/drivers/crypto/hisilicon/hisi_qm.c

```diff
@@ -646,24 +670,21 @@ static enum hisi_drv_status qm_cqc_cfg(struct hisi_qp *qp)
 {
 	enum hisi_drv_status ret = HISI_QM_DRVCRYPT_NO_ERR;
 	struct hisi_qm *qm = qp->qm;
-	struct qm_cqc *cqc = NULL;
-	paddr_t cqc_dma = 0;
-
-	cqc = memalign(HISI_QM_ALIGN32, sizeof(struct qm_cqc));
-	if (!cqc)
-		return HISI_QM_DRVCRYPT_ENOMEM;
-
-	cqc_dma = virt_to_phys(cqc);
+	struct qm_cqc *cqc = qm->cfg_xqc.cqc;
+	struct qm_mailbox mb;
```

Comment:
Please initialize this local variable with `= { }` as described in the coding guidelines.